Frontend/ui-customer/src/Components/EmergencyHelp/Controller.purs

```diff
@@ -25,23 +25,37 @@ import Styles.Colors as Color
 data Action = NoAction 
             | CallPolicePopup
             | ContactSupportPopup
+            | CallSuccessfulPopup
+            | CallContactPopUp Contact
             | CallPolice PopUpModalController.Action
             | ContactSupport PopUpModalController.Action
+            | CallEmergencyContact PopUpModalController.Action
+            | CallSuccessful PopUpModalController.Action
             | GenericHeaderAC GenericHeaderController.Action
 
 type EmergencyHelpModelState = {
```

Comment:
move this type to `Screen.Types` file and create a variable in `HomeScreenStateData` with this type and store the state of the component. Instead of creating multiple variable in `HomeScreenStateProps`.

Reply:
Done